src/collective/lazysizes/upgrades/v10/__init__.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+from collective.lazysizes.config import IS_PLONE_5
+from collective.lazysizes.logger import logger
+from plone import api
+
+
+NEW_JS = '++resource++collective.lazysizes/lazysizes.js'
+OLD_JS = '++resource++collective.lazysizes/lazysizes-umd.min.js'
+TWITTER_JS = '++resource++collective.lazysizes/ls.twitter.min.js'
+
+
+def use_webpack(setup_tool):
+    """Use resource compiled from webpack."""
+    if IS_PLONE_5:
+        logger.warn('Upgrade step not supported under Plone 5')
+        return
+
+    portal_js = api.portal.get_tool('portal_javascripts')
+    resource_ids = portal_js.getResourceIds()
+    if OLD_JS in resource_ids:
+        portal_js.renameResource(OLD_JS, NEW_JS)
+        logger.info('lazysizes was upgraded; using now webpack version')
+
+    assert OLD_JS not in portal_js.getResourceIds()
+    assert NEW_JS in portal_js.getResourceIds()
+
+
+def remove_twitter_plugin(setup_tool):
+    """Remove Twitter plugin."""
+    if IS_PLONE_5:
```

Comment:
upgrade steps will not supported under Plone 5 until the mess is fixed; just skip and warn users.

also, I don't see the poing on having 2 different methods here please merge them into one: remove older static files and add the new one.

Reply:
Done.